hoodie-client/src/test/java/com/uber/hoodie/io/strategy/TestHoodieCompactionStrategy.java

```diff
@@ -108,17 +109,25 @@ public void testPartitionAwareCompactionSimple() {
     sizesMap.put(110 * MB, Lists.newArrayList());
     sizesMap.put(100 * MB, Lists.newArrayList(MB));
     sizesMap.put(90 * MB, Lists.newArrayList(1024 * MB));
+    sizesMap.put(90 * MB, Lists.newArrayList(1024 * MB));
+    sizesMap.put(90 * MB, Lists.newArrayList(1024 * MB));
+    sizesMap.put(90 * MB, Lists.newArrayList(1024 * MB));
+
     DayBasedCompactionStrategy strategy = new DayBasedCompactionStrategy();
     HoodieWriteConfig writeConfig = HoodieWriteConfig.newBuilder().withPath("/tmp").withCompactionConfig(
-        HoodieCompactionConfig.newBuilder().withCompactionStrategy(strategy).withTargetIOPerCompactionInMB(400).build())
-        .build();
+        HoodieCompactionConfig.newBuilder().withCompactionStrategy(strategy)
```

Comment:
Not specific to this code change. But In general, its better to have deterministic unit-tests in cases which gives us the flexibility to add stronger assertions.